src/common/src/chirp.cpp

```diff
@@ -18,6 +18,8 @@
 #include "chirp.hpp"
 #include "debuglog.h"
 
+#define LIBUSB_ERROR_IO        -1
+#define LIBUSB_ERROR_NO_DEVICE -4
```

Comment:
Any reason these are not enums instead?

Reply:
Makes sense if it was a full overhaul of the pixy usb code but this is a hack/workaround to detect usb issues.